docs/src/examples/components/Table/BestPractices/TableBestPractices.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react'
+
+import { Text } from '@fluentui/react'
+import ComponentBestPractices from 'docs/src/components/ComponentBestPractices'
+import { link } from '../../../../utils/helpers'
+
+const doList = [
+  <Text>
+    Choose desired accessibility behavior for grid cell depending on the use case.(Check the{' '}
+    {link('Behaviors', '/components/table/accessibility')} section).
+  </Text>,
+  'Provide label to the Table component using `aria-label` or `aria-labelledby` prop.',
```

Comment:
pls add: `If no label is provided, the whole header will be read by the screen reader when the user navigates to the table`